src/sas/sascalc/dataloader/readers/cansas_reader_HDF5.py

```diff
@@ -2,11 +2,12 @@
     NXcanSAS data reader for reading HDF5 formatted CanSAS files.
 """
 
+import logging
 import h5py
 import numpy as np
 import re
 import os
-import sys
+import traceback
```

Comment:
pep8 suggests imports standard library imports come before third party packages, so h5py and numpy should drop below, with a blank line before.